components/server/src/prometheus-metrics.ts

```diff
@@ -151,11 +151,11 @@ export function observeAPICallsDuration(method: string, statusCode: number, dura
 const httpRequestTotal = new prometheusClient.Counter({
     name: "gitpod_server_http_requests_total",
     help: "Total amount of HTTP requests per express route",
-    labelNames: ["method", "route", "statusCode"],
+    labelNames: ["method", "route", "statusCode", "httpVersion"],
```

Comment:
This is working:
![Uploading image.png…]()

Reply:
Still working after changes:
![image](https://github.com/user-attachments/assets/0e88419e-3561-47c9-9f7c-4f1ae839a768)